modules/core/web/src/components/ModelingTool.vue

```diff
@@ -304,4 +358,31 @@ const calculateGradients = (groupedEntitys: Connection[][], calculatedLines: Lin
   cursor: default;
   z-index: 10;
 }
+
+.context-menu {
+  position: fixed;
+  background-color: white;
+  border: 1px solid #ccc;
+  z-index: 1000;
+  min-width: 100px;
+  max-width: 200px;
+  box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.2);
+  padding: 8px;
+  cursor: pointer;
+}
+
+.context-menu div {
+  padding: 5px;
+  cursor: pointer;
+}
+
+.context-menu-item {
+  padding: 4px 0;
+  transition: background-color 0.2s;
+  cursor: pointer;
+}
+
+.context-menu-item:hover {
+  background-color: #f0f0f0;
+}
```

Comment:
if the vue components are used it could be possible that this is no longer needed.